openpype/hosts/nuke/plugins/publish/extract_slate_frame.py

```diff
@@ -298,7 +298,10 @@ def _render_slate_to_sequence(self, instance):
 
     def add_comment_slate_node(self, instance, node):
 
-        comment = instance.data["comment"]
+        comment = (
+            instance.context.data.get("comment")
```

Comment:
I was wondering, if we have a different comment per instance and a context comment, the context comment would take over the one in the instance, right?